scripts/automation/trex_control_plane/interactive/trex/astf/trex_astf_profile.py

```diff
@@ -2026,6 +2030,9 @@ def __init__(self, default_ip_gen, default_c_glob_info=None, default_s_glob_info
         self.tg_name_to_id = collections.OrderedDict()
         self.cache = ASTFProfileCache(profile = self)
 
+        self.fields = {}
+        self.fields['rand_client_port'] = rand_client_port
```

Comment:
```suggestion
        self.fields = {'rand_client_port': rand_client_port}
```

or if we expected future changes here

```suggestion
        self.fields = {
          'rand_client_port': rand_client_port,
        }
```

Reply:
I think preparing for future changes is a good idea.